style.css

```diff
@@ -0,0 +1,581 @@
+body {
+  justify-content: center;
+  margin: 0;
+}
+
+nav {
+  background-color: #3c3a39;
+  padding: 25px;
+  color: #fff;
+}
+
+.nav-items {
+  display: none;
+}
+
+.navbar-innercontainer {
+  display: flex;
+  justify-content: space-between;
+}
+
+.nav-logo {
+  font-weight: 500;
+  font-size: large;
+  font-family: sans-serif;
+}
+
+.baricon {
+  width: 20px;
+  cursor: pointer;
+}
+
+.bar {
+  width: 100%;
+  background-color: #ffff;
+  padding: 1.5px;
+  margin-bottom: 4px;
+}
+
+.herosection {
+  background-color: #1c1a19;
+  padding: 25px;
+  height: 520px;
+  background-image: url("./Assets/images/backgroundimage.png");
+  background-repeat: no-repeat;
+  background-size: 80%;
+  background-position: right -128px bottom -51px;
+}
+
+.textcontainer {
+  margin-top: 25px;
+}
+
+.h5 {
+  color: #ff6b00;
+  font-size: 15vw;
+  width: 90%;
+  font-weight: 400;
+  margin-bottom: 5px;
+  font-family:
+    create "Lucida Sans",
+    "Lucida Sans Regular",
+    "Lucida Grande",
+    "Lucida Sans Unicode",
+    Geneva,
+    Verdana,
+    sans-serif;
+}
+
+.h6 {
+  font-family:
+    system-ui,
+    -apple-system,
+    BlinkMacSystemFont,
+    "Segoe UI",
+    Roboto,
+    Oxygen,
+    Ubuntu,
+    Cantarell,
+    "Open Sans",
+    "Helvetica Neue",
+    sans-serif;
+  font-size: 24px;
+  color: #ffff;
+  font-weight: 400;
+  margin: 5px;
+}
+
+.hero-paragraph {
+  color: #ffff;
+  font-size: 18px;
+  line-height: 26px;
+  width: 100%;
+}
+
+.icon-container {
+  display: flex;
+  align-items: baseline;
+  width: 60%;
+  padding: 0;
+}
+
+.icon_link {
+  margin: 0 7px;
+}
+
+li {
+  text-decoration: none;
+  list-style: none;
+}
+
+.icon-image {
+  width: 18px;
+  height: auto;
+  cursor: pointer;
+}
+
+/* MIDDLESECTION */
+
+.middlesection-container {
+  padding: 16px;
+  width: 90%;
+  text-align: center;
+  margin-top: 8%;
+}
+
+.middlesection_header_title {
+  font-size: 10vw;
+  margin-bottom: 15%;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  font-weight: 400;
+}
+
+hr {
+  color: #212121;
+  margin-bottom: 5%;
+}
+
+.middlesection-image {
+  margin-top: 7vh;
+  width: 100%;
+}
+
+.multiples-stories_container {
+  padding: 16px;
+  width: 90%;
+  margin-top: 8%;
+}
+
+.multiples-stories_header {
+  font-size: 10vw;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  font-weight: 400;
+  margin-top: 0;
+}
+
+.multiples-stories_paragraph {
+  color: #091e42;
+  font-size: 18px;
+  line-height: 26px;
+}
+
+.button-flex {
+  display: flex;
+  justify-content: space-between;
+  padding: 0;
+}
```

Comment:
- [ ] Kindly make sure that the lsit of technologie is using flex start so your design will be aligned with the [figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1-1471&t=yDKg5dMjRcmmZ2y5-0) 😄
**The figmat template isi actully using flex-start
![image](https://user-images.githubusercontent.com/74411135/227731657-b15535d7-331c-4fa3-9a9f-5e9120e735d0.png)